add information about development TLS (#267)
QwantResearch/erdapfel: README.md

**hint**: to debug the application from a browser, it is preferable to disable the dynamic loading of javascript files.
When starting the server, add `TILEVIEW_system_evalFiles=false` to your config.

#### Local node TLS errors (only for development and debug, **NOT** production!!!)

> For certain request to node server, you might see on node server logs:

Reviewer: ```suggestion
Some requests to node server could trigger this message on node server logs:
```

```text
"msg":"unable to get local issuer certificate"
```

You can overpass this error by setting `NODE_TLS_REJECT_UNAUTHORIZED=0` on the server environment such as:

```bash
> NODE_TLS_REJECT_UNAUTHORIZED=0 npm run start
```

> Again, **NEVER** use this option for something else than development!!!

Reviewer: ```suggestion
Note that you are **NEVER** supposed to use this option for anything else than development.
```
